src/components/form/checkbox/_checkbox.scss

```diff
@@ -3,6 +3,10 @@
 
   &:not(.euiCheckbox--noLabel) .euiCheckbox__input {
     @include euiScreenReaderOnly;
+    // Because this input is still focusable, it causes the screen-reader outline
+    // to extend to the position of the screen-reader element.
+    // So we must position it against it's own bounds.
+    left: 0;
   }
```

Comment:
Actually!! I have an idea for you that I _think_ might work across both Safari & Firefox in VO. I'd like to propose ditching the `euiScreenReaderOnly` mixin for the input **entirely**, and instead doing what the `noLabel` version of the input is doing: resizing and overlaying the hidden input on top of the visual element. This solves the outline matching the element in all browsers as far as I can tell, and we can also lose the new `:not()` selector this way (since it's basically the same CSS for both labelled and un-labelled inputs).

Here's the new CSS:

```suggestion
  // Floats above the visual checkbox and matches its dimension so that
  // when no label is present, users will actually click this input
  .euiCheckbox__input {
    position: absolute;
    left: 0;
    top: (($euiSizeL - $euiCheckBoxSize) / 2) - 1px;
    @include size($euiCheckBoxSize); /* 1 */
    opacity: 0;
    z-index: 1;
    margin: 0;
    cursor: pointer;
  }
```

and the CSS for the noLabel input simply becomes:
```scss
  &.euiCheckbox--noLabel {
    // ...

    .euiCheckbox__input {
      top: auto;
      outline: none;
    }
```

Here's a commit you can check out with the diff: https://github.com/constancecchen/eui/commit/142c1699c4caab1078e7857d71d3a8cb75b1a92f

Let me know what you think!